gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Updated `checksum_algorithm` plugin with a permanent fix to pass JRuby CI.
```

Comment:
```suggestion
* Issue - Updated `checksum_algorithm` plugin to use IO.copy_stream for JRuby.
```

Reply:
Thats not really what its doing though - its just fixing a default argument that was missing in the plugin so that the way our specs work don't need special casing for JRuby (the actual JRuby net::http client worked fine with this code, so I don't believe this was an actual issue, just an artifact of the way these specs were written).